web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
@lasley Indeed it isn't, but it's in our mind. Here is the PR: https://github.com/OCA/maintainer-tools/pull/217